libs/sdk-backend-base/src/cachingBackend/index.ts

```diff
@@ -380,6 +449,11 @@ export type CachingConfiguration = {
      * @param cacheControl - cache control instance
      */
     onCacheReady?: (cacheControl: CacheControl) => void;
+
+    /**
+     * TODO js doc when optimal values are found
+     */
+    maxSecuritySettings: number | undefined;
```

Comment:
i'm contemplating whether it makes more sense to have just a boolean here to turn caching of security settings on/off. the size limitation for other types of caches is in place because their contents can be massive.

additionally, if we stay with the numeric setting, we should have two & do some renames. first-level is number of cached scopes and the other is number of cached urls. these can have different cardinalities (typically working in one scope and possibly validating more URLs)

Reply:
I went with second option. One number to setup number of cached organizations, second to specify how many URLs can be cached per organization, and third one to specify maximum age of each URL response in milliseconds.